packages/language-support/src/autocompletion/completionCoreCompletions.ts

```diff
@@ -51,6 +51,22 @@ function backtickDbNameIfNeeded(e: string): string | undefined {
   }
 }
 
+function backtickParamIfNeeded(e: string): string | undefined {
+  if (e == null || e == '') {
+    return undefined;
+  }
+  const namePart = e.slice(1);
+  if (e[0] == '$') {
```

Comment:
Should I even have this test, or just trust the backtickParam... -method to only be called with params?

Reply:
I think it's okay to have double guards in case something goes wrong in the other part of the code.